### PR TITLE
Fixed url checking code in LoadUrl()

### PR DESCRIPTION
**Pull request by JongHeonChoi**

### CODE REVIEW

File: runtime/browser/web_view_impl.cc [hunk 1]
```diff
@@ -100,12 +100,13 @@ void WebViewImpl::LoadUrl(const std::string& url, const std::string& mime) {
...
   if (!mime.empty()) {
     mime_ = mime;
+    url_ = url;
     auto mime_override_cb = [](const char* url, const char* mime,
                                char** new_mime, void* data) -> Eina_Bool{
       WebViewImpl* view = static_cast<WebViewImpl*>(data);
       if (view != nullptr &&
           common::utils::BaseName(url) ==
-            common::utils::BaseName(view->GetUrl())) {
+            common::utils::BaseName(view->url())) {
```
Comment (WonyoungChoi) [inline runtime/browser/web_view_impl.cc]: url_ is used in only this line. why don't you use url parameter directly instead of adding url() setter/getter.

**Comment by WonyoungChoi** [comment]
Please add more description about the reason of this commit in the commit message.

File: runtime/browser/web_view_impl.cc (hunk 1 above)
Comment (JongHeonChoi) [inline runtime/browser/web_view_impl.cc]: build error : url is not captured for lambda function

File: runtime/browser/web_view_impl.h [hunk 2]
```diff
@@ -80,6 +79,8 @@ class WebViewImpl {
   void InitRotaryEventCallback();
 #endif  // ROTARY_EVENT_FEATURE_SUPPORT
 
+  std::function<bool (const char*, const char*, char**, void*)> mime_set_cb;
```
Comment (WonyoungChoi) [inline runtime/browser/web_view_impl.h]: mime_set_cb_
Add underscore suffix for private member variable

File: runtime/browser/web_view_impl.cc [hunk 3]
```diff
@@ -96,25 +96,31 @@ WebViewImpl::~WebViewImpl() {
...
-void WebViewImpl::LoadUrl(const std::string& url, const std::string& mime) {
+void WebViewImpl::LoadUrl(const std::string& first_url,
+                          const std::string& wanted_mime) {
   SCOPE_PROFILE();
-  if (!mime.empty()) {
-    mime_ = mime;
-    auto mime_override_cb = [](const char* url, const char* mime,
-                               char** new_mime, void* data) -> Eina_Bool{
+  if (!wanted_mime.empty()) {
+    auto mime_set_cb = [first_url, wanted_mime]
```
Comment (WonyoungChoi) [inline runtime/browser/web_view_impl.cc]: use member variable, not declared variable in here

File: runtime/browser/web_view_impl.cc [hunk 4]
```diff
@@ -96,25 +96,31 @@ WebViewImpl::~WebViewImpl() {
     evas_smart_free(evas_smart_class_);
 }
 
-void WebViewImpl::LoadUrl(const std::string& url, const std::string& mime) {
+void WebViewImpl::LoadUrl(const std::string& first_url,
+                          const std::string& wanted_mime) {
```
Comment (WonyoungChoi) [inline runtime/browser/web_view_impl.cc]: rename pameter name of lamba instead

**Comment by myroot** [comment]
LGTM

**Pull request merged**